File: Semestru 4/MPP/lab/.net/lab1/lab1/Models/Domain/Child.cs
namespace lab1.Models.Domain
{
	public class Child : Entity<int>
	{
        public string Name { get; set; }
        public string Cnp { get; set; }

        public Child(string Name, string Cnp)
        {
            this.Name = Name;
            this.Cnp = Cnp;
        }

        public Child() { }
    }
}

File: Semestru 4/MPP/lab/java/lab3/Model/src/main/java/org/example/domain/Child.java
package org.example.domain;

public class Child extends Entity<Integer>{
    private String Name;
    private String CNP;

    public Child (String name, String cnp) {
        Name = name;
        CNP = cnp;
    }

    public String GetName() {
        return this.Name;
    }

    public String GetCNP() {
        return this.CNP;
    }

    @Override
    public String toString() {
        return this.Name + ' ' + this.CNP;
    }
}
